scripts/dev_setup.sh

```diff
@@ -95,11 +100,17 @@ function install_build_essentials {
 }
 
 function install_rustup {
+  echo installing rust.
   BATCH_MODE=$1
   # Install Rust
-  [[ "${BATCH_MODE}" == "false" ]] && echo "Installing Rust......"
-  if rustup --version &>/dev/null; then
-	   [[ "${BATCH_MODE}" == "false" ]] && echo "Rust is already installed"
+  if [[ "${BATCH_MODE}" == "false" ]]; then
+     echo "Installing Rust......"
+  fi
+  VERSION="$(rustup --version || true)"
```

Comment:
Not sure if i see the need for or-ing here.  If rustup is not installed, the env var will be empty and satisfies the check below.

Reply:
sub shell will fail if rustup isn't there.   resulting in an error.